ghga_event_schemas/__init__.py

```diff
@@ -15,4 +15,4 @@
 
 """A package that collects schemas used for events exchanges between GHGA service."""
 
-__version__ = "0.6.1"
+__version__ = "0.6.2"
```

Comment:
Mein vorheriger PR hat schon auf 0.7.0 gebumpt:

```suggestion
__version__ = "0.7.1"
```